questions.md

```diff
@@ -0,0 +1,4 @@
+# Question on Pull Request
+1. Please what is the difference between git merge &  pull request.
+2. In what scenario can we use both.
```

Comment:
Git merge you might use for individual work. A pull request you might use for group work :)